### ISSUE TITLE
Media Keys disabling

### ISSUE THREAD

**Issue by xdubx** [MatMoul/g810-led#247]
Hi,

have Pop OS 20.04. The problem is after the systemd services started the OS don't recognize the media keys any more and the volume slider on my Keyboard is the g815.

**Comment by MatMoul** [comment]
g810-led is a one shot command and don't handle your keys...
How do you have installed it?

**Comment by xdubx** [comment]
clone repo, make files. install.

**Comment by xOchoa** [comment]
I'm getting the same issues, none of the media keys works after install

**Comment by MatMoul** [comment]
No idea, tagging to need investigation

**Comment by KiddingMe42** [comment]
Same issue with Ubuntu 18.04 (g213) - also the lighting is turned off after restart and can only enabled after plug the usb in again
xev nor showkeys -k can detect the media keys after make install
I removed the tool with make uninstall - reboot - and now it is working again.

**Comment by saracalihan** [comment]
Same issue. OS: `5.10.18-1-Manjaro`, keyboard: `Logitech G G213 Prodigy`

**Comment by saracalihan** [comment]
> Same issue. `5.10.18-1-Manjaro`, `Logitech G213 Prodigy`

I uninstall the project then install `g810-led-lib` from `aur` then the media keys works.

**Comment by DarioMarzella** [comment]
Same issue here, Ubuntu 20.04.02, Keyboard G910.

**Comment by AMDBartek** [comment]
Same issue on Kubuntu 20.10 with a Logitech G810 keyboard.

**Comment by zocker-160** [comment]
the issue is when g810-led is using the USB interface, because when using libusb, the media keys are connected to the same endpoint and when g810-led requests access to said endpoint, it will unload the kernel driver, which handles the media keys.

The solution is to use hidapi, which does not need exclusive access to endpoints like libusb does.

**Comment by MPatzek** [comment]
Is there a way to install hidapi mode via apt-install on ubuntu (mint) or is a manual installation needed?

**Comment by zocker-160** [comment]
> Is there a way to install hidapi mode via apt-install on ubuntu (mint) or is a manual installation needed?

Hidapi mode needs to be supported by g810-led, installing additional packages will not fix the issue.

**Comment by MPatzek** [comment]
> 

Yes, that's clear. But there is an option for hidapi. Maybe I should rewrite my questions:
Can I install g810-led via apt-get using the hidapi mode such as g810-led-hidapi? (similar to compiling by "mkae bin" without lsusb)
If not, I need to uninstall, and run the manual installation by cloning the repo and buiilding it using hidapi.

**Comment by zocker-160** [comment]
Ah ok now I understand what you mean, sorry.

IIRC the Ubuntu repository has only one package available. If that is using libusb, then you will need to manually compile it.
